src/components/List/List.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import classnames from 'classnames';
+import * as styles from './List.sass';
+import Header from '../Header/Header';
+import IReactComponentProps from '../../common/structures/IReactComponentProps';
+
+const fontSizeContentClassMixin = (styles2: {[key: string]: any}, props: {[key: string]: any}) => ({
```

Comment:
Does TypeScript throw an error here if you use `styles` instead of `styles2`?

Reply:
Yes. Somewhat similar to `index2`. I think it's just a matter of figuring out naming in these situations.